tools/wrapper_common/execute.py

```diff
@@ -64,7 +66,12 @@ def execute_and_filter_output(cmd_args,
       stdout=subprocess.PIPE,
       stderr=subprocess.PIPE,
       env=env)
-  stdout, stderr = proc.communicate(input=inputstr)
+  try:
+      stdout, stderr = proc.communicate(timeout=timeout)
```

Comment:
@brentleyjones maybe because of missing `input` here?

Reply:
Yep, that was it! PR incoming.